packages/core/src/tools/mcp-tool.test.ts

```diff
@@ -743,4 +743,13 @@ describe('DiscoveredMCPTool', () => {
       }
     });
   });
+
+  describe('DiscoveredMCPToolInvocation', () => {
+    it('should return the stringified params from getDescription', () => {
+      const params = { param: 'testValue' };
+      const invocation = tool.build(params);
+      const description = invocation.getDescription();
+      expect(description).toBe(JSON.stringify(params));
```

Comment:
nit: can one of the tests show the actual output rather than verifying it matches what was computed. Also include multiple params to make sure it is human readable enough.